docs/UserGuide.adoc

```diff
@@ -109,15 +119,15 @@ Now that you have understood the command format, let's explore the features.
 
 === Managing your contactHeRo account
 
-This section describes the commands which enable to you use the login system to secure your data stored in contactHeRo.
+This section describes the commands you can use to secure your data stored in contactHeRo by the login system.
```

Comment:
This sounds incorrect. Data is not stored by the login system. Discard these changes.

Reply:
change to "This section describes the commands which enable you to use the login system to secure your data stored in contactHeRo."